app/components/Index/common/constants.ts

```diff
@@ -53,24 +55,29 @@ const {
   FILES,
   SPECIES,
   SPECIMENS,
+  TOTAL_FILE_SIZE,
 } = SUMMARY;
 
 /**
  * Functions binding summary response API to summary count.
  */
-export const BIND_SUMMARY_RESPONSE = {
-  [BIOSAMPLES]: (r: AzulSummaryResponse): number =>
-    getSummaryCount(r, SUMMARY_KEY.BIOSAMPLES),
-  [DONORS]: (r: AzulSummaryResponse): number =>
-    getSummaryCount(r, SUMMARY_KEY.DONORS),
-  [ESTIMATED_CELLS]: calculateSummaryTotalCellCount,
-  [FILES]: (r: AzulSummaryResponse): number =>
-    getSummaryCount(r, SUMMARY_KEY.FILES),
-  [FILE_FORMATS]: calculateSummaryFileFormatsCount,
-  [SPECIES]: (r: AzulSummaryResponse): number =>
-    getSummaryCount(r, SUMMARY_KEY.SPECIES),
-  [SPECIMENS]: (r: AzulSummaryResponse): number =>
-    getSummaryCount(r, SUMMARY_KEY.SPECIMENS),
+export const BIND_SUMMARY_RESPONSE: {
+  [key in SUMMARY]: (r: AzulSummaryResponse) => string;
+} = {
+  [BIOSAMPLES]: (r: AzulSummaryResponse): string =>
+    formatCountSize(getSummaryCount(r, SUMMARY_KEY.BIOSAMPLES)),
```

Comment:
`formatCountSize` added to each binder fn to allow `totalFileSize` to be formatted by `formatFileSize`. See https://github.com/DataBiosphere/data-browser/pull/4571/files#r2408973202.